Models/Contact.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhoneBookApp.Models
+{
+    public class Contact
+    {
+        [Required]
+        public int Id { get; set; }
+
```

Comment:
Name and PhoneNumber should be required. It doesn't make sense for a record not to have one of them in a phone book application.